### PR TITLE
README.md: suggested changes

### PR DESCRIPTION
**Pull request by grahamperrin**

### CODE REVIEW

**Comment by grahamperrin** [comment]
As requested. 

The currently suggested changes probably go considerably beyond what you might expect from proofreading; sorry! 

Some of the most significant edits were, partly, for me to try to understand what's going on. Some guesswork, because I have not attempted to run the script (and I don't know scripting, I'm not a developer). 

If this guesswork and my general lack of knowledge about the boot process have resulted in suggestions that are contentious or totally incorrect: similarly, my apologies. 

(Whilst I am aware of [The FreeBSD Boot Process - Klara Systems](https://klarasystems.com/articles/the-freebsd-boot-process/) (2022), and related manual pages, I simply don't understand various things …)

**Comment by grahamperrin** [comment]
### Side note

I don't know whether you're familiar with review features in GitHub. If not, this might help: 

- [Révision des changements proposés dans une demande de tirage - Documentation GitHub](https://docs.github.com/fr/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request)

Maybe most useful, this feature: 

image

**Comment by Emrion** [comment]
Many thanks for your review. I will take time to evaluate your suggestions, but I can already say that a good part is ok for me. It's just the beginning that troubles me a little. And, yes, I know almost nothing about github.

File: README.md [hunk 1]
```diff
@@ -1,40 +1,61 @@
...
-What it does handle:
-- Checks all the disks reported by the system (case of mirror disks).
-- EFI and BIOS loaders.
-- EFI partitions mounted or not mounted.
-- Checks for the presence of EFI loaders in efi/ and lists them all.
-- Recognizes (or try) to identify FreeBSD EFI loaders and would update only them.
-- Doesn't change the name of EFI FreeBSD loaders (case where the admin would have changed the default ones).
-- In case of freebsd-boot partition, checks the coherence between its content and the root file system.
-- Checks if each detected loader is already up to date and doesn't try or propose to update in this case.
+`loaders-update` is a proof of concept, a [sh(1)](https://man.freebsd.org/cgi/man.cgi?query=sh&sektion=1&manpath=freebsd-release) script for attention to:
```
Comment (Emrion) [inline README.md:3]:
```suggestion
`loaders-update` is a tool, a [sh(1)](https://man.freebsd.org/cgi/man.cgi?query=sh&sektion=1&manpath=freebsd-release) script for attention to: 
```

File: README.md [hunk 2]
```diff
@@ -1,40 +1,61 @@
...
+The script can help to update these things.
+
+Some people feel that automation is not possible, or not desirable.
+Too complex, they say … so, I wrote this script.
+
+### FreeBSD Project changes to the loader
+
+Sometimes, these changes relate to changes to ZFS.
+
+Somtimes not. An outdated loader might be problematic _without_ changes to ZFS; _without_ an upgrade to a ZFS pool; or with _UFS_ on root.
```
Comment (Emrion) [inline README.md]:
```suggestion
Sometimes not. An outdated loader might be problematic _without_ changes to ZFS; _without_ an upgrade to a ZFS pool; or with _UFS_ on root. 
```

**Comment by Emrion** [comment]
I tried to make some reviews on your "pull request", but I'm a kinda lost with github. I would like to simplify all the beginning and just leave a sparce description of this tool goals. May I commit your "pull request" and commit my changes just after? I feel It will be easier. That said, I don't even know how to commit your "pull request", lol.

**Review by Emrion — APPROVED** [review]

**Pull request merged**